lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -468,7 +468,7 @@ def store_at_0
           end
 
           def update_positions
-            old_position = send("#{position_column}_was").to_i
```

Comment:
As I wrote below, the only reason I can think of, is to prevent errors in `shuffle_positions_on_intermediate_items` when old_position is nil